### ISSUE TITLE
Very high average loss

### ISSUE THREAD

**Issue by amankagarwal** [AlexeyAB/darknet#1412]
I am trying to train to detect 2 classes, using around 130 images. I am getting an average loss of about 700-1000, which seems very high. Screenshot and CFG file below. I am training on an NVIDIA GTX 1050 if that helps. Any help is appreciated!:

![cmd](https://user-images.githubusercontent.com/14858512/44148164-610b3d0c-a097-11e8-872b-1a156f5a4cc3.PNG)
```
[net]
# Testing
batch=64
subdivisions=64
# Training
# batch=64
# subdivisions=16
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1
```

**Comment by sonalambwani** [comment]
Typically, the loss starts off high but falls off to single digits within a couple thousand iterations. These are my observations from running a yolo3 training on own dataset. My training data is ~100,000 images.

How many iterations in are you? I think you should try lowering the batch size. Also, are you using the pre-trained weights from darknet?

**Comment by amankagarwal** [comment]
Yes, I am using pre-trained weights- https://pjreddie.com/media/files/darknet53.conv.74 
I am only ~100 iterations in right now, but I tried running to about 1000 iterations last night (took about 8 hours) on a different dataset but still had an average loss of about 1000.

**Comment by amankagarwal** [comment]
The average loss did start going down as @sonalambwani mentioned. At 230 iterations it's in single digits now.
